de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ProjectNegotiation.java

```diff
@@ -37,7 +37,7 @@
     /**
      * Prefix part of the id used in the SMACK XMPP file transfer protocol.
      */
-    public static final String ARCHIVE_TRANSFER_ID = "saros-dpp-pn-server-client-archive/";
+    public static final String TRANSFER_ID = "saros-dpp-pn-server-client-archive/";
```

Comment:
TRANSFER_ID_PREFIX or something like that and I would also remove / replace archive in the value of the string.